index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mrprotocoll Portfolio</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+
+    <nav class="navbar">
+        <div class="container nav">
+            <div class="logo"><a href="#">My Logo</a></div>
+            <div class="bars">
+                <div class="bar-item"></div>
+                <div class="bar-item"></div>
+                <div class="bar-item"></div>
+            </div>
+        </div>
+    </nav>
+
+    <header class="hero container">
+        <div class="hero-content">
+            <h1>I’m Dami <br> Glad to see  <br> you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="cta"><h3>LET’S CONNECT</h3></a>
+            <ul class="social">
+                <li class="icon"><a href="#"><img src="assets/images/twitter.svg" alt=""></a></li>
+                <li class="icon"><a href="#"><img src="assets/images/linkedin.svg" alt=""></a></li>
+                <li class="icon"><a href="#"><img src="assets/images/microverse.svg" alt=""></a></li>
+                <li class="icon"><a href="#"><img src="assets/images/github.svg" alt=""></a></li>
+                <li class="icon"><a href="#"><img src="assets/images/vector.svg" alt=""></a></li>
+            </ul>
+        </div>
```

Comment:
- [ ] Kindly use` section tag `to wrap all these elements, a `section` is a semantic element for creating standalone sections in a web page. It's more preferable that `div tag` to group similar elements. ℹ️

Reply:
Thank you @OyePriscilla. I actually used the `nav` and `header` semantics to group each navbar and hero section. I didn't think it is necessary to use a section inside those semantics. I will make the changes and create another pull request.

Reply:
Done! Kindly check if it is ok with you now @OyePriscilla. so I can proceed to submit a new review.

Reply:
> Thank you @OyePriscilla. I actually used the `nav` and `header` semantics to group each navbar and hero section. I didn't think it is necessary to use a section inside those semantics. I will make the changes and create another pull request.

@mrprotocoll It is awesome! Actually `section` is semantic but `div `has no meaning.